pages/integrations/segment.mdx

```diff
@@ -0,0 +1,32 @@
+---
+layout: 'guide'
+title: 'Segment integrations'
+---
+
+import Link from '../../../components/Link';
+
+## Stream Segment data to Inspector
+
+<Link passHref href="/workspace/connect-inspector-to-segment"><a>Stream Segment data to Inspector</a></Link>
+
+## Send app data to Inspector with the SDK
+
+<Link passHref href="/implementation/avo-inspector-sdk-reference"><a>Stream app data to Inspector with an SDK</a></Link>
+
+## Publish tracking plan to Protocols
+
+<Link passHref href="/workspace/tracking-plan/publishing#segment-protocols"><a> Publish tracking plan to Protocols</a></Link>
+
+## Generate code to send data to Segment
```

Comment:
Here we should be highlighting the value props of each integration point

Reply:
Do you suggest doing it in this pr? Not sure what exactly to write here

Reply:
We should do it before we start multiplying this doc for other integration types. We can ship this now and iterate before we write integration docs for other destinations, or hold off to merging until we've nailed down the copy here